subsys/net/lib/openthread/platform/diag.c

```diff
@@ -17,18 +15,22 @@
  */
 static bool sDiagMode;
 
-void otPlatDiagProcess(otInstance *aInstance,
-		       int argc,
-		       char *argv[],
-		       char *aOutput,
-		       size_t aOutputMaxLen)
+otError otPlatDiagProcess(otInstance *aInstance,
+					uint8_t argc,
```

Comment:
Please fix the alignment. Note, it's 8 spaces per tab, I guess hence the confusion.